source/viz.md

```diff
@@ -160,7 +160,7 @@ understand and remember your message quickly.
 +++
 
 ## Creating visualizations with `altair`
-#### *Build the visualization iteratively*
+#### Build the visualization iteratively
```

Comment:
No sure why this used to be in italics

Reply:
@joelostblom the other sub-sub-sub-sub section headings in this chapter are also italicized, so this would introduce an inconsistency.

But I think there's a bigger problem -- the translation from R to Python broke things here anyway.

The point of these subheadings wasn't to introduce a new numerical section, it was just to have a "punchline" for each section. 

- One option is to remove this from both versions. 
- The other option is to keep it in both but make sure it renders properly in the Py version -- not as a subsection

R version:

![image](https://github.com/UBC-DSCI/introduction-to-datascience-python/assets/5981307/7894e1af-f5e2-40e3-a313-5baa71010a91)

Py version:

![image](https://github.com/UBC-DSCI/introduction-to-datascience-python/assets/5981307/d5f5a2d2-a45c-4dd1-9b04-8d2f574b2dcb)

Reply:
My bad, I missed that somehow. I can fix the subheading numbering in another PR; it's straightforward from what I recall